integration-cli/check_test.go

```diff
@@ -38,8 +38,12 @@ func (s *DockerRegistrySuite) SetUpTest(c *check.C) {
 }
 
 func (s *DockerRegistrySuite) TearDownTest(c *check.C) {
-	s.reg.Close()
-	s.ds.TearDownTest(c)
+	if s.reg != nil {
```

Comment:
Nice catch :+1: . During my development of this PR, this piece of code paniced because one of my bug.
I would remove this to reduce confusion and focus more on the golint warning removement.